### PR TITLE
Container systemd fixes

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

* Proxy: add image tag configuration to systemd service
* Add comments to the config file and improve the README.md
* Remove `for` loops in `%service_{add|del}_{pre|post}` calls
  
## GUI diff

No difference.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- No tests: work in progress

- [X] **DONE**

## Links

Fixes https://github.com/SUSE/spacewalk/issues/17788

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

**Review by ncounter — APPROVED** [review]
:+1:

**Comment by cbosdo** [comment]
@juliogonzalez could you review the `spec` file part of that PR?

**Comment by cbosdo** [comment]
No real need to run acceptance tests on that PR since it changes code that is NOT tested by it at all

File: containers/proxy-systemd-services/uyuni-proxy-systemd-services.spec [hunk 1]
```diff
@@ -63,51 +63,51 @@ install -D -m 644 uyuni-proxy-services.config %{buildroot}%{_fillupdir}/sysconfi
...
 done
 
 %check
 
 %pre
 %if !0%{?rhel}
-for service in %{SERVICES}; do
-    %service_add_pre uyuni-${service}.service
-done
+%service_add_pre %{SERVICES}
```
Comment (juliogonzalez) [inline containers/proxy-systemd-services/uyuni-proxy-systemd-services.spec:74]: As long as you checked RHEL systems supports the services in one line, I am fine with it.

**Review by juliogonzalez — APPROVED** [review]
Looks OK

File: containers/proxy-systemd-services/uyuni-proxy-systemd-services.spec (hunk 1 above)
Comment (juliogonzalez) [reply containers/proxy-systemd-services/uyuni-proxy-systemd-services.spec:74]: Sorry, my comment above is wrong. I wrote it, then noticed my comment was wrong... but didn't remember to remove it before sending the review.

**Pull request merged**